packages/keystone/src/lib/migrations.ts

```diff
@@ -252,7 +252,7 @@ We need to reset the ${credentials.type} database "${credentials.database}" at $
       console.log(`✨ A migration has been created at migrations/${generatedMigrationName}`);
 
       let shouldApplyMigration =
-        migrationCanBeApplied && (await confirmPrompt('Would you like to apply this migration?'));
```

Comment:
@mitchellhamilton I have defaulted this to `no`,  this should help prevent an unintentional `DROP`.